packages/server-socket/src/command-processor.spec.js

```diff
@@ -36,6 +36,19 @@ describe("command-processor", () => {
       });
     });
 
+    it("sends a CHANGED_TODO event when a correct CHANGE_TODO command was received", () => {
+      const { processCommand } = createCommandProcessor({ Event, logger: console });
+      const command = { command: "CHANGE_TODO", data: { title: "Change a title", todoId: "id-1" } };
+      processCommand(sendEvent)(command).then(x => {
+        expect(createMock).toHaveBeenCalled();
+        expect(sendEvent).toHaveBeenCalledTimes(1);
+        expect(createMock.mock.calls[0][0]).toMatchSnapshot({
+          createdAt: expect.any(Number)
+        });
+        expect(createMock.mock.calls[0][0]["eventType"]).toEqual("CHANGED_TODO");
```

Comment:
Should not be necessary if snapshot is tested before

Reply:
probably not, that's true

Reply:
on the other hand, it makes it quite obvious what's being tested... which i think is a good thing